examples/ssr/src/request.ts

```diff
@@ -0,0 +1,8 @@
+import lightcookie from 'lightcookie';
+
+
+export function isLoggedIn(request: Request): boolean {
```

Comment:
nit: can we call this file `util.ts` so that its clear its not some special "magic" src file?

Reply:
Why would you think that this file is magic? Is it the name `request`? I'm not a huge fan of the "util.ts" pattern as I think it encourages putting a bunch of unrelated things into the same file. 

If it's just about the name `request` I can rename it to something else. This is really model logic, maybe `models/user.ts`?

Reply:
I moved this code to models/user.ts.